### PR TITLE
feat(FX-4174): Implement standalone page for Activity Panel (Scaffolding)

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Feat**

This PR solves [FX-4174]

Review app: https://standalone-notifications.artsy.net/notifications

### Description
* Implement a new Activity Panel standalone page. This page will be shown **only for mWeb** users currently.
* New activity panel should be hidden behind an Unleash feature flag. It is enough to hide only “Activity” link in a menu behind the feature flag (https://github.com/artsy/force/pull/10827)
* It’s OK to not account for desktop UI. There will be no direct link to this page in desktop, only in the mobile version.
* When clicking on a notification - redirect user to a “targetURL” from a Metaphysics response.
* DO NOT implement tabs and pagination yet - just render the full list of notifications. Improvements will be done in next tickets.
* DO NOT implement “mark as read” functionality yet - will be done later
* Design components in a way that it can be possible to reuse them when implementing Activity Panel Dropdown (for desktop).

### Demo
| mWeb |
| ------------- |
|  |

[FX-4174]: https://artsyproduct.atlassian.net/browse/FX-4174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dd074ef3e2b49afd0b9109e6c6782d16eaac0cd0
</p>

File: src/Apps/Notifications/notificationsRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  () =>
+    import(/* webpackChunkName: "notificationsBundle" */ "./NotificationsApp"),
+  {
+    resolveComponent: component => component.NotificationsApp,
+  }
+)
+
+export const notificationsRoutes: AppRouteConfig[] = [
+  {
+    path: "/notifications",
```
Comment (dimatretyak) [inline src/Apps/Notifications/notificationsRoutes.ts:14]: **question**: do you think such a path is suitable or is it better to use another one (for example, `/activities`)?

File: src/Server/middleware/userRequiredMiddleware.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ const USER_REQUIRED_ROUTES = [
   "/settings/payments",
   "/settings/purchases(.*)",
   "/settings/shipping",
+  "/notifications",
```
Comment (dimatretyak) [inline src/Server/middleware/userRequiredMiddleware.ts:10]: if a user who is not logged in tries to open the page, we will redirect to the log in page

Comment (damassi) [reply src/Server/middleware/userRequiredMiddleware.ts:10]: alphabetical pls 🙏

Comment (dimatretyak) [reply src/Server/middleware/userRequiredMiddleware.ts:10]: Done

File: src/Server/middleware/userRequiredMiddleware.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { match } from "path-to-regexp"
 
 const USER_REQUIRED_ROUTES = [
+  "/notifications",
```
Comment (dimatretyak) [inline src/Server/middleware/userRequiredMiddleware.ts:4]: if a user who is not logged in tries to open the page, we will redirect to the log in page

**Review by dblandin — APPROVED** [review]
LGTM! Nice use of alt and aria tags! 💯

**Comment by nickskalkin** [comment]
Wondering why text massage is misaligned with object_ids count, for example: "53 works added", but only +6 displayed on images rail. It's not frontend issue, I will look into backend

**Pull request merged**